src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { setPaymentTokenFetch } from '../../../payment-methods/cybersource/service';
+
+class FlexMicroform extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            microform: null,
+        }
+        this.onLoad = this.onLoad.bind(this);
+        this.getPaymentToken = this.getPaymentToken.bind(this);
+    }
+
+    componentDidMount() {
+        setPaymentTokenFetch(this.getPaymentToken);
+
+        const script = document.createElement('script');
+        script.src = "https://flex.cybersource.com/cybersource/assets/microform/0.11/flex-microform.min.js";
+        script.async = true;
+
+        script.addEventListener('load', this.onLoad);
+        script.addEventListener('error', e => {
+            console.error(e);
+        });
+
+        document.body.appendChild(script);
+    }
+
+    initialize() {
+        if (this.state.microform !== null) {
+            return;
+        }
+        if (!this.props.captureKeyId) {
+            return;
+        }
+        if (window['Flex'] === undefined) {
+            return;
+        }
+        const microform = new window.Flex(this.props.captureKeyId).microform({
+            styles: { 
```

Comment:
hack: hardcoded style

Reply:
This may be hard to fix for now since we don't have our style constants expressed as JS - just in CSS.  

If we later switch to some CSS-in-JS, keeping this in sync with our theme will be a lot easier.